src/lib/ydocker/main_dialog.rb

```diff
@@ -18,6 +18,7 @@
 
 require "yast"
 require "docker"
+require "y2storage"
```

Comment:
as it use only DiskSize, does it make sense to require only this? this module is quite lightweight and requiring whole storage stack looks quite big for me.